doc/api/documentation.md

```diff
@@ -28,11 +28,9 @@ The stability indices are as follows:
 
 <!-- separator -->
 
-> Stability: 1 - Experimental. This feature is still under active development
-> and subject to non-backward compatible changes or removal in any future
-> version. Use of the feature is not recommended in production environments.
-> Experimental features are not subject to the Node.js Semantic Versioning
-> model.
+> Stability: 1 - Experimental. The feature is not subject to Semantic Version
```

Comment:
s/Version/Versioning/?

Reply:
Oh, yeah, that would be better. Will make that change right now. Thanks.